warehouse/query-core/src/test/java/datawave/query/language/parser/jexl/TestLuceneToJexlQueryParser.java

```diff
@@ -567,5 +585,20 @@ public void testParseRegexsWithEscapes() throws ParseException {
         Assert.assertEquals("FOO == 'bar' && BAZ =~ 'Foo/Foo\\ Foo.*'", parser.parse("FOO:bar BAZ:/Foo\\/Foo\\ Foo.*/").getOriginalQuery());
         Assert.assertEquals("FOO == 'bar' && BAZ =~ 'Foo/Foo Foo.*?'", parser.parse("FOO:bar BAZ:/Foo\\/Foo Foo.*?/").getOriginalQuery());
     }
+
+    private static class TestQueryNodeProcessorFactory extends QueryNodeProcessorFactory {
+        @Override
+        public QueryNodeProcessor create(QueryConfigHandler configHandler) {
+            return new QueryNodeProcessorPipeline(configHandler);
+        }
+    }
+
+    @Test
```

Comment:
Demonstrates how `LuceneToJexlQueryParser` behavior changes after injecting an alternate `QueryNodeProcessorFactory` implementation. In this case, it uses the default lucene `QueryNodeProcessorPipeline` that does not perform field tokenization in the same way as the datawave `CustomerQueryNodeProcessorPipeline`.